recipe/meta.yaml

```diff
@@ -158,6 +158,7 @@ outputs:
         - xz
         - zstd
         - openssl  # exact pin handled through openssl run_exports
+        - {{ pin_compatible('poppler', max_pin='x.x') }}
```

Comment:
`poppler` is mentioned twice in `run`. The [first one](https://github.com/AnacondaRecipes/gdal-feedstock/pull/26/files#diff-f3725a55bf339595bf865fec73bda8ac99f283b0810c205442021f29c06eea9aR154) is redundant

Reply:
We might have to pin to `x.x.x` since they link against `libpoppler.so` which doesn't have a stable ABI (https://poppler.freedesktop.narkive.com/bW6aTH5Z/abi-stability#post2).